.gitignore

```diff
@@ -10,6 +10,7 @@ tests/integration/*-tester/lib/
 .env
 cos-tool*
 *.egg-info
+.charm_tracing_buffer.raw
```

Comment:
What's that?
Does it have to reside in the charm dir?
I.e. if this is a reasonable issue to raise on the charm tracing lib then we probably shouldn't have it here?

Reply:
After checking with the tracing team, I added a fixture to move that file somewhere else.